features/frameworks/nanostack-libservice/mbed-client-libservice/ns_types.h

```diff
@@ -117,10 +115,7 @@ typedef int_fast32_t int_fast24_t;
 
 /* C11's "alignas" macro, emulated for integer expressions if necessary */
 #ifndef __alignas_is_defined
-#if defined __CC_ARM || defined __TASKING__
```

Comment:
This should be
```
#if defined __TASKING__
```

Reply:
[Done](https://github.com/ARMmbed/mbed-os/compare/ec33c1d353fb7ad7b6300ec060f781f22f54b50d..209f1e6a7190323202b9f9049230307ff8e9a387).